katharsis-ui/src/index.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```

Comment:
Obviously this UI isn't ready to go. But, it's a good start. How will you discover the resources available?